src/D2L.CodeStyle.UnsafeStaticCounter/Properties/AssemblyInfo.cs

```diff
@@ -10,5 +10,5 @@
 
 [assembly: ComVisible( false )]

```

Comment:
Yeah I saw that recently (assuming you're talking about the`assembly_info` rule in the yaml). I can switch it after you do Core.